src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -122,6 +122,51 @@ export default class ReimbursementRequestService {
     return createdReimbursementRequest;
   }
 
+  /**
+   * Function to reimburse a user for their expenses.
+   *
+   * @param recipientId the user who will be receiving the reimbursement
+   * @param amount the amount to be reimbursed
+   * @param submitter the person performing the reimbursement
+   * @returns the created reimbursement
+   */
+  static async reimburseUser(amount: number, submitter: User): Promise<Reimbursement> {
+    if (!isAdmin(submitter.role)) {
```

Comment:
users don't need to be admins to submit a reimbursement, since we're just having ppl submit their own reimbursements. Just check that the user isn't a guest instead (update the corresponding test)